src/media.js

```diff
@@ -831,7 +808,7 @@
           // console.log('AUDIO' + allAudioArr[158].length);
           allAudioSend = [];
           audioLen = 0;
-          for (luid in audioToBePlay) {
+          for (letluid in audioToBePlay) {
```

Comment:
Codacy found an issue: [for..in loops iterate over the entire prototype chain, which is virtually never what you want. Use Object.{keys,values,entries}, and iterate over the resulting array.](https://app.codacy.com/gh/congrea/virtualclass/pullRequest?prid=6361181)